include/ShaderWriter/Intrinsics/IntrinsicFunctions.hpp

```diff
@@ -8,7 +8,7 @@ This file is generated, don't modify it!
 #define ___SDW_IntrinsicFunctions_H___
 #pragma once
 
-#include "ShaderWriter/BaseTypes/Bool.hpp"
+#include "ShaderWriter/BaseTypes/Boolean.hpp"
```

Comment:
You need to modify the script that generates this file.
(In scripts/ShaderWriter/)